### PR TITLE
Real trending

### PR DESCRIPTION
**Pull request by YuanxunTao**
approving

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -22,21 +22,21 @@ MongoClient.connect('mongodb://localhost', { useNewUrlParser: true }).then(conne
...
-//TODO: GET should return top x number of videos with highest upvotes-downvotes number
 
 //GET VIDEOS ENDPOINT
 app.get('/api/videos/:id', (req,res) => {
   let wikiId = parseInt(req.params.id,10); //converting from string to number
   
   db.collection('videos').find({wikiPageId: wikiId}).count()
   .then(count => {
-    if(count == 0)  res.json(constructResponse(true,`No videos for wikiPage: ${wikiId} found!`, {videos: [], count:count})); 
+    if(count == 0) res.json(constructResponse(true,`No videos for wikiPage: ${wikiId} found!`, {videos: [], count:count}));
```
Comment (sjakati98) [inline server.js:32]: this space was integral to the operation of the application

File: server.js [hunk 2]
```diff
@@ -136,6 +159,37 @@ app.put('/api/videos',(req,res) => {
 
 // Helpers/Constants
 
+const getSectionIds = (videos) => {
```
Comment (aasnani) [inline server.js:162]: I claim ownership of these changes

File: src/ContentPage.jsx [hunk 3]
```diff
@@ -62,16 +62,17 @@ const Video = (props) => (
 
 const AddVideoModal = (props) => (
   <div>
-    <button type="button" className="btn btn-primary" data-toggle="modal" data-target="#AddVideoModal">Add Video</button>
-
+    <div id="buttonOnRight">
```
Comment (aasnani) [inline src/ContentPage.jsx:65]: I like when things are right

File: src/SearchPage.jsx [hunk 4]
```diff
@@ -1,53 +1,21 @@
...
-}
 
 class TrendingCard extends React.Component {
   render() {
     const item = this.props.item;
+    const yturl = 'https://youtube.com/watch?v=' + item.ytId;
+    const ytimg = 'http://i3.ytimg.com/vi/' + item.ytId + '/hqdefault.jpg';
     return (
-      <div className="col">
+      <div className="col" style={{width: "33%"}}>
```
Comment (aasnani) [inline src/SearchPage.jsx:11]: 100 doesn't cleanly divide into 33, anxiety.

**Review by seangtkelley — APPROVED** [review]

File: src/ContentPage.jsx (hunk 3 above)
Comment (sjakati98) [reply src/ContentPage.jsx:65]: This comment LEFT me feeling RIGHT on the inside

File: src/SearchPage.jsx [hunk 5]
```diff
@@ -153,8 +103,31 @@ let TitleLogo = (props) => (
...
+    this.loadRendingData = this.loadRendingData.bind(this);
+  }
+
+  componentDidMount() {
+    this.loadRendingData();
+    console.log("component mounted");
+    console.log(this.state.trendingData);
+  }
+
+  loadRendingData() {
```
Comment (aasnani) [inline src/SearchPage.jsx:117]: trigger

**Review by sjakati98 — APPROVED** [review]
Neat.

File: static/style.css [hunk 6]
```diff
@@ -30,4 +30,11 @@
 
 #howItWorksList{
     text-align: justify;
+}
+
+#buttonOnRight{
+    display: flex;
```
Comment (aasnani) [inline static/style.css:36]: flex on the display B-)

**Review by aasnani — APPROVED** [review]

File: static/style.css (hunk 6 above)
Comment (sjakati98) [reply static/style.css:36]: Use emojis its 2017 bro 😎

**Pull request merged**